python/tests/test_gunicorn_utils.py

```diff
@@ -0,0 +1,51 @@
+import os
+import signal
+import time
+from typing import Set
+
+import pytest
+import requests
+from prometheus_client.parser import text_string_to_metric_families
+
+
+def _get_workers(
+    metrics_endpoint: str = "http://127.0.0.1:6005/metrics-endpoint",
```

Comment:
Where does port `6005` comes from?

Reply:
That's defined in `python/tests/helpers.py`